triage/script.js

```diff
@@ -0,0 +1,738 @@
+"use strict";
+
+var rightArrow = "\u25ba";
+var downArrow = "\u25bc";
+
+var builds = null;
+var clustered = null;		  // filtered clusters
+var clusteredAll = null;	  // all clusters
+var options = null;			  // user-provided in form or URL
+var lastClusterRendered = 0;  // for infinite scrolling
+
+const kGraphHeight = 200; 	  // keep synchronized with style.css:div.graph
+
+/**********************************************************************
+ *                        POLYFILLS                                   *
+ **********************************************************************/
+
+if (Object.entries === undefined) {
+	// Simple polyfill for Safari compatibility.
```

Comment:
"In web development, a polyfill is code that implements a feature on web browsers that do not support the feature. "